lib/error_message.ex

```diff
@@ -127,7 +172,7 @@ defmodule ErrorMessage do
   """
   defdelegate to_jsonable_map(error_message), to: ErrorMessage.Serializer
 
-  @spec inspect(error_message :: t) :: String.t
+  @spec inspect(error_message :: t) :: String.t()
```

Comment:
Sorry, this was done by my formatter, can revert it.

Reply:
Revert this one please! 😉 My libs are always non formattered